CONTRIBUTING.md

```diff
@@ -0,0 +1,114 @@
+# Contributing
+
+This document provides guidelines for contributing to this repository.
+
+## Development Environment Setup
+
+### Prerequisites
+* Linux machine with x86-64 architecture with bash installed.
+* Basic tools: curl, tar, make.
+
+### Running the Automated Tests
+1. Clone your fork of the project.
+2. Download dependencies and run automated tests: `make all`.
+
+## Automated Tests
+The following tools are currently used for automated testing within the project:
+* [promtool](https://prometheus.io/docs/prometheus/latest/configuration/unit_testing_rules/) - testing PromQL rules, queries and PromQL linting
+* [pint](https://cloudflare.github.io/pint/checks/promql/series.html) - PromQL linting
+* [gitlint](https://jorisroovers.com/gitlint/) - commit message linting
+* [yamllint](https://yamllint.readthedocs.io/en/stable/) - YAML linting
```

Comment:
The URL here can be replaced with `https://yamllint.readthedocs.io` which redirects to https://yamllint.readthedocs.io/en/stable/ anyway